src/server/snapshot.h

```diff
@@ -164,8 +164,11 @@ class SliceSnapshot : public journal::JournalConsumerInterface {
 
   // Used for sanity checks.
   bool serialize_bucket_running_ = false;
+
+  std::string snapshot_fb_name_;
```

Comment:
helio fiber stores a string view, previously it pointed just to a stack placed string 😳

Reply:
Are you sure it's needed?